src/main/java/org/truffleruby/language/SafepointManager.java

```diff
@@ -140,10 +140,15 @@ private SafepointAction step(Node currentNode, boolean isDrivingThread) {
         return deferredAction;
     }
 
+    private static final int WAIT_TIME = 5; // seconds
```

Comment:
I'd rather embed the unit in the variable name. It makes the name longer, but I think it's worth that small expense.

Reply:
Done. I wish there was a Duration class to complement TimeUnit but there is only java.time's one it seems which is much more involved and does not use TimeUnit.